### ISSUE TITLE
Client crashing on bootup 'net.minecraft.class_2535 failed'

### ISSUE THREAD

**Issue by TheSnowcraft** [QuiltMC/quilt-loader#412]
I've been using Quilt for a small while now without issues. Unfortunately, that all recently changed when I started working on my own mod, (which I've not even compiled yet). I decided to pause on working on it to play a little in my ongoing survival world, when I discovered my client was now crashing on startup, despite not changing my mod list. The error 'net.minecraft.class_2535 failed' stood out to me when reading through the crash log (provided below).

[crash-2024-03-14_00.39.27-client.txt](https://github.com/QuiltMC/quilt-loader/files/14595240/crash-2024-03-14_00.39.27-client.txt)

I'm deeply confused, and a little concerned, since this only started happening after I installed a Java Development Kit to begin working on my own mod, and would like to find a solution that doesn't involve me having to stop working on my own mod project.

**Comment by OroArmor** [comment]
Appears to be an issue with "allthetrims". I would make sure you have the right and latest version for your minecraft version

**Comment by TheSnowcraft** [comment]
So it is. I seem to have the correct version (3.4.2 Fabric 1.20.1), so I'm not sure what's going on there. Maybe a specific thing it doesn't like with Quilt?

**Comment by OroArmor** [comment]
Ah ok it appears to be using fabric internals for some reason. That would cause the issue.

**Comment by TheSnowcraft** [comment]
Right. So one of those 1% of mods that just isn't compatible with Quilt for obscure technical reasons?

**Comment by eristhea** [comment]
if I'm reading the code right it does a mixin into FAPI implementation classes to improve error handling? seems like it should be easy for the dev to disable the mixin on quilt
I may make a PR later
